interface-definitions/protocols-isis.xml.in

```diff
@@ -3,9 +3,9 @@
 <interfaceDefinition>
   <node name="protocols">
     <children>
-      <tagNode name="isis" owner="${vyos_conf_scripts_dir}/protocols_isis.py">
```

Comment:
Why is this renamed to is-is? All Vendors I know call it isis, also FRR. We should stick to the industry standard.